select_statements.sql

```diff
@@ -31,89 +31,60 @@ UPDATE vehicle
-SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori, v.stationName as Station
-FROM vehicle as v
-INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-ORDER BY Station, FordonsKategori;
-
-
 -- Select all vacant cars for a station
 SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori
-FROM vehicle as v
+FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-WHERE v.stationName = "Uppsala station" AND (b.endDate < "2024-04-13" OR b.endDate IS NULL)
-ORDER BY Fordonskategori;
+WHERE v.stationName = "Uppsala station" AND (b.endDate < "2024-04-13" OR b.endDate IS NULL);
 
 -- Select all vacant cars for all stations
 SELECT v.registrationNumber as Registreringsnummer, vc.name as Fordonskategori, v.stationName as Station
-FROM vehicle as v
+FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
-WHERE b.endDate < "2024-04-13" OR b.endDate IS NULL
-ORDER BY Station, Fordonskategori;
+WHERE b.endDate < "2024-04-13" OR b.endDate IS NULL;
 
 -- Select all vacant cars for "my" station of a certain category
-SELECT v.registrationNumber, v.stationName FROM vehicle as v
+SELECT v.registrationNumber, v.stationName FROM green_rental.vehicle as v
 LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber
 LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber
 INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId
 WHERE v.stationName = "Uppsala station" AND vc.name = "Stadsbil" AND (b.endDate < "2024-04-12" OR b.endDate IS NULL);
 
--- Select number of cars per station
-SELECT v.stationName as Station, count(*) as "Antal bilar" 
-FROM vehicle as v GROUP BY v.stationName;
-
--- Select all stations with vacant cars
-SELECT v.stationName as Station, count(*) as "Antal lediga bilar" 
-FROM vehicle as v 
-INNER JOIN booking as b ON v.stationName = b.stationName 
-WHERE v.registrationNumber IN 
-	(  
-		SELECT v.registrationNumber 
-        FROM vehicle as v  
-        LEFT JOIN booking_details as bd ON v.registrationNumber = bd.registrationNumber  
-        LEFT JOIN booking as b ON bd.bookingNumber = b.bookingNumber  
-        INNER JOIN vehicle_category as vc ON v.vehicleCategoryId = vc.vehicleCategoryId  
-        WHERE b.endDate < "2024-04-12" OR b.endDate IS NULL 
-	) 
-GROUP BY v.stationName;
-
-
-
-
--- Underhållspersonal
 -- Sök fram alla bilar i behov av kontroll
-SELECT vehicle.registrationNumber FROM vehicle
+SELECT booking_details.bookingNumber as bokningsnummer, vehicle.registrationNumber as registreringsnummer, 
```

Comment:
Do we need the bookingNumber in the result? Also, you can use " to create a resulting field which is easier to read: "Senaste kontrolldatum".